package.json

```diff
@@ -18,14 +18,16 @@
   },
   "scripts": {
     "watch": "yarn install-if-deps-outdated && ./node_modules/.bin/gulp watch",
-    "clean-client": "rm -rf _inc/build/ _inc/blocks/ css/",
+    "clean": "yarn clean-client && yarn clean-extensions",
+    "clean-client": "rm -rf _inc/build/ css/",
+    "clean-extensions": "rm -rf _inc/blocks/ ",
     "install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files",
-    "distclean": "rm -rf node_modules",
-    "build": "yarn install-if-deps-outdated && yarn clean-client && yarn build-client && yarn build-extensions",
+    "distclean": "rm -rf node_modules && yarn clean",
+    "build": "yarn install-if-deps-outdated && yarn clean && yarn build-client && yarn build-extensions",
```

Comment:
Just noting that using [`run-s`](https://www.npmjs.com/package/npm-run-all) or Gulp tasks we could run the `build-client` and `build-extensions` in parallel, speeding things up a bit.

That's for another PR tho.